make/doc.mk

```diff
@@ -52,15 +52,17 @@ else
 DOC_TARGETS?=html
 endif
 
+EX_DOC_WARNINGS_AS_ERRORS?=true
```

Comment:
Last question, should this be false by default?

I immediately had to turn it off in the nix build. Which worked by the way! Here is the simplification I was able to do with this https://github.com/chiroptical/nixpkgs/pull/4/files.

Reply:
No I want it to be on by default as there should never be any warnings if you use the correct ExDoc vsn.

Reply:
I get this from the build with that set to true.

```
warning: the VM is running with native name encoding of latin1 which may cause Elixir to malfunction as it expects utf8. Please ensure your locale is set to UTF-8 (which can be verified by running "locale" in your shell) or set the ELIXIR_ERL_OPTIONS="+fnu" environment variable
```

I guess you may want to set this in the wrapper then?

Reply:
That would be something that needs fixing in nix I think. The Erlang VM thinks that filenames should be encoded in latin1 because the locale is not set to utf8.

The elixir nix package seems to set [LANG and LC_TYPE](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/interpreters/elixir/generic-builder.nix#L37-L38) to `C.UTF-8` which should do the trick.

Reply:
I guess so! Really appreciate you looking through all this.